main.html

```diff
@@ -162,9 +171,6 @@
     margin-top: 2.29rem;
     margin-bottom: 0.89rem;
   }
-  h3 > code {
-    margin-left: -0.1rem;
```

Comment:
This is now lost. It was here to ensure a better visual left alignment, to correct for the inner padding and light background of the `<code>`.